src/content/index.js

```diff
@@ -88,6 +90,8 @@ function observeBody() {
         addPlayerProfileDownloadDemo(modalElement)
         addProfileMatchesEloPoints(modalElement)
         addPlayerProfileExtendedStats(modalElement)
+      } else if (modals.isPlayerProfile()) {
+        addPlayerProfileBadge(modalElement)
```

Comment:
I'm sorry, I didn't realize that it was actually correct when it was nested, so `addPlayerProfileBadge` is only called once when it's the player profile in general. Would you mind reverting this back to the previous state please.